### DISCUSSION TITLE
Funnel analysis

### DISCUSSION THREAD

**Discussion by MarkusAtEnreach** [plausible/analytics#1755] [Anything Else]
Hi,

how to use plausible for funnel analysis? Is there any training material (video, blog, documentation) available?

Regards,
Markus

UPDATE: Both [funnels](https://plausible.io/docs/funnel-analysis) and [user journeys](https://plausible.io/docs/user-journeys) are now available in Plausible!

**Comment by metmarkosaric** [comment]
hi @MarkusAtEnreach!

funnel visualization is something we plan to add but cannot make any promise on when. we are a small team with limited development resources so need to focus on one thing at a time. there are a few other more frequently requested features we have to complete first (currently working on GA stats import) before being able to start the work on this.

[our custom events](https://plausible.io/docs/custom-event-goals) are acceptable workaround at the moment. even when we introduce funnels, they will be based on custom events. you'll be allowed to choose which custom events are part of a funnel and which event is what step of a funnel. so all the data that we will show in funnels is there already as part of our current custom events and you can see it in the current dashboard. what you're missing is a nice visual chart that shows these different custom events in the correct order and with drop off percentages between them, you would need to manually calculate these now.

only new thing we might introduce with funnels is for them to be based on session tracking. for instance, if you're tracking a signup process with a natural order that all visitors go through, session based tracking won't make any difference compared to how custom events work now. but if you're tracking more random things that any visitor can start at any random step, then session based tracking would not consider those but will only consider those visitors that start with the correct step that you selected and that go through the funnel as you've set it up in the correct order. that will remove some of the noise but again not really important for signup processes and similar which is what most people use funnels for

i hope this helps! do stay tuned!

**Reply by richardsorensson** [reply]
as it's possible to create sites, events and goals via the api it'd also be great to create the funnels via the api once they are introduced.

**Comment by metmarkosaric** [comment]
**Update**: We've now opened up a preview version of funnels to everyone. You can now follow the visitor journey from a landing page to a conversion with funnel analysis. Details: https://plausible.io/docs/funnel-analysis

**Reply by Eyesonly88** [reply]
will funnels be available in the open-source version? v2.0?

**Reply by oliverpalonkorp** [reply]
Also wondering this. Very useful

**Reply by plantoeat** [reply]
I am unable to create a funnel. The page keeps reloading every second. We are currently on the 10M pageviews plan, will you charge extra for funnels?

**Reply by metmarkosaric** [reply]
Hi @plantoeat! Could you please try and clear cookies for Plausible.io and then log in again? Then funnels should work all fine. Thanks!

**Reply by plantoeat** [reply]
Clearing cookies worked! Will funnels be an extra cost on top of the current monthly fee?

**Reply by metmarkosaric** [reply]
great, glad to hear that @plantoeat! and yes, these new features (funnels and revenue goals) will become a part of our new business plans later this year. they’re free-to-use during the preview but will be exclusive to business plan subscribers from october. more details and pricing will be announced in the upcoming weeks

**Reply by alekseyp** [reply]
What about self-hosted users?

**Reply by troyeisert** [reply]
Are there any plans to provide an API endpoint to get data for an entire funnel?

**Comment by metmarkosaric** [comment]
UPDATE: Both [funnels](https://plausible.io/docs/funnel-analysis) and [user journeys](https://plausible.io/docs/user-journeys) are now available in Plausible!
